front/src/common/redux/auth/auth.saga.js

```diff
@@ -15,8 +16,38 @@ function* loginSaga(payload) {
     }
 }
 
+function* changePasswordSaga(payload) {
+    try {
+        if (payload === 'is success') {
```

Comment:
Why do we need this check?

Reply:
I use it for changing state after success
I call Spinner when isChangedPassword is true, and after notification I need to have isChangedPassword: false, because notification will show after each updating